packages/gafl-webapp-service/src/pages/licence-details/licence-start-time/route.js

```diff
@@ -6,33 +6,55 @@ import moment from 'moment-timezone'
 import { cacheDateFormat } from '../../../processors/date-and-time-display.js'
 import { nextPage } from '../../../routes/next-page.js'
 
-const minHour = permission =>
-  moment(permission.licenceStartDate, cacheDateFormat)
+const getMinHour = permission => {
+  const now = moment().tz(SERVICE_LOCAL_TIME)
+  const permissionStartsToday = moment(permission.licenceStartDate, cacheDateFormat)
     .tz(SERVICE_LOCAL_TIME)
-    .isSame(moment().tz(SERVICE_LOCAL_TIME), 'day')
-    ? moment()
-      .tz(SERVICE_LOCAL_TIME)
-      .add(1, 'hour')
-      .add(30, 'minute')
-      .startOf('hour')
-      .hour()
-    : 0
-
-const hours = Array(24)
+    .isSame(now, 'day')
+  if (permissionStartsToday) {
+    const cantStartUntilTomorrow = moment(now).add(90, 'minute').isAfter(now, 'day')
+    if (cantStartUntilTomorrow) {
+      return 24
+    }
+    return now.add(90, 'minute').startOf('hour').hour()
+  }
+  return 0
+}
+
+const hours = Array(25)
```

Comment:
How come this has changed to 25?

Reply:
Yeah, we need to handle 10:31pm onwards on a day, you'll have seen the `cantStartUntilTomorrow` in minHour, which is for when we've passed the cut off for the day. The minHour function will then return '24', which isn't an available option, so there is no valid option.